config/initializers/curation_concerns_actors.rb

```diff
@@ -0,0 +1,2 @@
+CurationConcerns::Actors::ActorFactory.remove_actor CurationConcerns::Actors::AddToCollectionActor
```

Comment:
Yeah, it's on master over in ualbertalib/hydranorth2. I just dumped the contents of a bunch of branches into master so I can plan out what needs to happen next sprint. Sorting out this cross-repo testing is part of it.